src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -70,16 +71,26 @@ public function evaluate(array $functions, array $values)
             case self::PROPERTY_CALL:
                 $obj = $this->nodes['node']->evaluate($functions, $values);
                 if (!\is_object($obj)) {
+                    if (is_null($obj) && $this->nodes['attribute']->isNullSafe) {
+                        return null;
+                    }
                     throw new \RuntimeException(sprintf('Unable to get property "%s" of non-object "%s".', $this->nodes['attribute']->dump(), $this->nodes['node']->dump()));
                 }
 
                 $property = $this->nodes['attribute']->attributes['value'];
 
+                if (!property_exists($obj, $property)) {
```

Comment:
we should remove this since now it's unrelated (and might break dynamic properties implemented with __get())